niceman/resource/docker_container.py

```diff
@@ -42,6 +42,8 @@ class DockerContainer(Resource):
         doc="Docker base image ID from which to create the running instance")
     engine_url = attrib(default='unix:///var/run/docker.sock',
         doc="Docker server URL where engine is listening for connections")
+    seccomp = attrib(default='on',
+        doc="Disable kernel secure computing mode when creating the container")
```

Comment:
With the doc saying "Disable kernel secure computing mode ..." I don't think it's clear that `default=on` means the code does *not* disable it.

Maybe make `seccomp_unconfined` the attribute (defaulting to `False`)?

Reply:
Yes, your suggestion is more clear and intuitive. Made the change.